ui/src/pages/Lore/Lore.tsx

```diff
@@ -31,32 +32,39 @@ export function Lore() {
     };
     fetchData();
   }, []);
-  console.log(lore[currentLoreSelection])
+
+  const isToggled = () => {
+    setToggled(!toggled);
+  }
+  console.log(toggled)
   return (
     <div className="lore">
       {loading ? (
         <> loading </>
       ) : (
-      <div className="lore__container">
-      <div id="fade"></div>
-        <div className="lore__container__effect">
-          <div className="lore__container__effect__text">
-            <h3 className="header3">LOCATION:</h3>
-            <p className="paragraph">{lore[currentLoreSelection].location}</p>
-            <h3 className="header3">Character:</h3>
-            <p className="paragraph">{lore[currentLoreSelection].character}</p>
-            <h3 className="header3">Background:</h3>
-            {sectionedBackground.map((section) => (
-              <div key={section}>
-                <p className="paragraph">
-                  {section}
-                </p>
-                <br />
+        <>
+          <button type="button" onClick={isToggled}>PRESS TO CHANGE</button>
+          <div className={toggled ? "lore__no_effect" : "lore__container"}>
+            {toggled ? null : <div id="fade"></div>}
+            <div className={toggled ? "lore__no_effect" : "lore__container__effect"}>
+              <div className={toggled ? "lore__no_effect" : "lore__container__effect__text"}>
```

Comment:
these should either be in a variable or in a function. Keep this turnary in a var

Reply:
how should i deal with the different layers of `lore__container`? like i can make a `var text = () => {isPlainText ? 'lore__no_effect' : "lore__container"}` but the more specific layers is what i am confused about. `lore__container__effect` and `lore__container__effect__text`